src/audio/audio.cpp

```diff
@@ -368,6 +368,35 @@ void Audio::playGroupAudioQueued(void*,int group, int peer, const int16_t* data,
     emit static_cast<Core*>(core)->groupPeerAudioPlaying(group, peer);
 }
 
+/**
+Must be called from the audio thread, plays a group call's received audio
+*/
+void Audio::playGroupAudio(int group, int peer, const int16_t* data,
+                           unsigned samples, uint8_t channels, unsigned sample_rate)
+{
+    assert(QThread::currentThread() == audioThread);
+    QMutexLocker locker(&audioLock);
+
+    if (!CoreAV::groupCalls.contains(group))
+        return;
+
+    ToxGroupCall& call = CoreAV::groupCalls[group];
+
+    if (call.inactive || call.muteVol)
+        return;
+
+    qreal volume = 0.;
+    int bufsize = samples * 2 * channels;
+    for (int i = 0; i < bufsize; ++i)
+        volume += abs(data[i]);
+
+    emit groupAudioPlayed(group, peer, volume / bufsize);
```

Comment:
Hm… not sure if we actually need this event anymore - to me it looks wrong. I'll do some more research and see what it was actually used for.